src/components/shared/Slider/index.tsx

```diff
@@ -1,20 +1,41 @@
-import React from 'react';
+import React, { FC } from 'react';
+import { PanResponder } from 'react-native';
 import styled from 'styled-components/native';
 
-const Container = styled.View`
-  flex: 1;
-  background-color: transparent;
-  flex-direction: row;
-  align-items: center;
-  justify-content: center;
+const Container = styled.View``;
+
+const Rail = styled.View`
```

Comment:
How about separate [Rail] component to code independently, may be in a sub directory?
That makes easy to test, manage and collaborate with our team with less conflicts.
It can be assembled into [Slider] component with the other components like [Track], [Label] and [Thumb].
If it needs to be here, we can move it into [Slider] after we implement each components.

Reply:
Yes, It's good.